### PR TITLE
Add debug data to unhandled exception stack dump in cnex

### PR DESCRIPTION
**Pull request by gitlarryf**

### CODE REVIEW

File: exec/cnex/cnex.c [hunk 1]
```diff
@@ -331,8 +337,83 @@ void exec_raiseLiteral(TExecutor *self, TString *name, TString *info, Number cod
...
+            if (debug != NULL) {
+                fread(debug, 1, nSize, fp);
+                fclose(fp);
+                free(dbgpath);
+                symbols = cJSON_Parse(debug);
+                // Remember the start of the symbols, so we can always go back to them.
+                pStart = symbols;
+                free(debug);
+            }
+        }
```
Comment (ghewgill) [inline exec/cnex/cnex.c]: Memory leak of `dbgpath` in the case where the file can't be found.

File: exec/cnex/cnex.c [hunk 2]
```diff
@@ -331,8 +337,83 @@ void exec_raiseLiteral(TExecutor *self, TString *name, TString *info, Number cod
...
+                lines = cJSON_GetObjectItem(symbols, "line_numbers");
+                cJSON_ArrayForEach(symbols, lines)
+                {
+                    jip = cJSON_GetArrayItem(symbols, 0);
+                    if ((uint64_t)jip->valueint == p) {
+                        jln = cJSON_GetArrayItem(symbols, 1);
+                        break;
+                    }
+                }
+                if ((uint64_t)jip->valueint == p) {
```
Comment (ghewgill) [inline exec/cnex/cnex.c]: Can you just check for `jln != NULL` here instead of repeating the equality test?

Comment (gitlarryf) [reply exec/cnex/cnex.c]: Fixed.

File: exec/cnex/cnex.c (hunk 1 above)
Comment (gitlarryf) [reply exec/cnex/cnex.c]: Fixed.

**Pull request merged**